charts/spire/charts/spiffe-csi-driver/templates/daemonset.yaml

```diff
@@ -79,7 +87,7 @@ spec:
             - name: kubelet-plugin-registration-dir
               mountPath: /registration
           ports:
-            - containerPort: 9809
+            - containerPort: {{ .Values.nodeDriverRegistrar.healthPort }}
```

Comment:
why make this configurable. Just adds to the complexity of the chart, where probably no one ever has to change this.

Reply:
This port unfortunately conflicts with AWS fargate on our clusters, and has to be changed for our usage.